### DISCUSSION TITLE
Participant simulation in Hubs local instance

### DISCUSSION THREAD

**Discussion by rishivinod7** [mozilla/hubs#3633] [General Questions & Discussion] [answered]
I'm running hubs locally and want to test some of my work which requires two participants, I run an instance on two different browsers. But, I can't seem to successfully share or join the host room, the share link pages do not work. I've tried "https://dev.reticulum.io/link" and "https://hub.link/" pages . Even tried embedding the html code given.  If anyone knows a working method to simulate a participant running locally, I would appreciate the help very much.

**Comment by robinkwilson** [comment, marked as answer]
Try copying the url from the address bar and pasting into both browser windows.

I run 2 browser windows with `https://hubs.local:8080/cVyRTQQ/fussy-ornate-territory` (This url will not work for yours) my locally generated room url and can see two of me OR `https://localhost:8080/cVyRTQQ/fussy-ornate-territory`

REMEMBER: if you're using the same browser, in the first tab, within 10 seconds you have to click the "Cancel" button to stop the tab from auto ending the session because it sees that you are in 2 rooms

**Reply by robinkwilson** [reply]
Marking this as answered for now, unmark as answered with more info if you're still having this issue!

**Reply by rishivinod7** [reply]
Hi thanks! It seems to be working again, I'll update this thread if there's any issue 🙂
